src/parsing/index.ts

```diff
@@ -136,3 +141,15 @@ export function parsePostgreSqlWithoutCursor(query: string): ParseResult {
     // In order to truly complete a finished query, we need to add space to it like so "SELECT * FROM |".
     return parsePostgreSql(query + ' ', '');
 }
+
+
+export function parseClickhouse(queryBeforeCursor: string, queryAfterCursor: string): ParseResult {
+    let parser = clickhouseAutocompleteParser as Parser;
+    return parser.parseSql(queryBeforeCursor, queryAfterCursor);
+}
+
+export function parseClickhouseWithoutCursor(query: string): ParseResult {
+    // If our finished query is "SELECT * FROM|" and we try to parse it, parser thinks that we still haven't finished writing it and doesn't show some errors.
+    // In order to truly complete a finished query, we need to add space to it like so "SELECT * FROM |".
+    return parseClickhouse(query + ' ', '');
```

Comment:
Let's move `query + ' '` to a separate function. We're copypasting it in 3 places